### PR TITLE
[E2E Tests] Camel Endpoint messaging tests

### PR DESCRIPTION
**Pull request by jsolovjo**
In this PR I added:
- Camel Endpoint messaging E2E tests (send, browse, forward, view)
- Fix a test method when expanding an item in the Camel tree (if already expanded - do not click again)

### CODE REVIEW

File: tests/hawtio-test-suite/src/main/java/io/hawt/tests/features/pageobjects/fragments/camel/tabs/endpoints/CamelSend.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+     * Add one header to the message.
+     *
+     * @param header      name to be added
+     * @param headerValue to be added
+     */
+    public void addOneHeader(String header, String headerValue) {
+        $(byTagAndText("button", "Add Headers")).shouldBe(enabled).click();
+        $(byAttribute("aria-label", "Search input")).shouldBe(enabled).click();
+        $(byTagAndText("p", header)).shouldBe(visible).click();
+        $(byXpath("//input[@aria-label='value-input-0']")).shouldBe(enabled).setValue(headerValue);
```
Comment (tadayosi) [inline tests/hawtio-test-suite/src/main/java/io/hawt/tests/features/pageobjects/fragments/camel/tabs/endpoints/CamelSend.java:27]: If this is not yet just defined on the element, consider defining some more meaningful value to it for hawtio-next.
> aria-label='value-input-0'

**Review by tadayosi — APPROVED** [review]

File: tests/hawtio-test-suite/src/main/java/io/hawt/tests/features/pageobjects/fragments/camel/tabs/endpoints/CamelSend.java (hunk 1 above)
Comment (jsolovjo) [reply tests/hawtio-test-suite/src/main/java/io/hawt/tests/features/pageobjects/fragments/camel/tabs/endpoints/CamelSend.java:27]: Ok, thanks for pointing to this. I will fix it

Comment (tadayosi) [reply tests/hawtio-test-suite/src/main/java/io/hawt/tests/features/pageobjects/fragments/camel/tabs/endpoints/CamelSend.java:27]: Once you send a pull req to hawtio-next, I can quickly release a new version of it so you can use it here.

Comment (jsolovjo) [reply tests/hawtio-test-suite/src/main/java/io/hawt/tests/features/pageobjects/fragments/camel/tabs/endpoints/CamelSend.java]: I have fixed and it seems all checks have passed. 
If it's Ok now, can you merge it, please?
Thanks

**Review by tadayosi — APPROVED** [review]

**Pull request merged**